duckduckgo/macos-browser
# Setup new Bundle ID + Ship Review feedback

Task/Issue URL: https://app.asana.com/0/0/1205440531945663/f


**Description**:
This PR setups the new bundle ID, adds new icon for DBP and address the following feedback from ship review:
1 - Remove Dax onboarding from DBP builds
2- Rename FAQ to Debug
3 - Change text location on name form from the footer to the header

**Steps to test this PR**:
1. Check if you compile the DBP target you'll see the new icon
2. Check if we don't have the check for updates button
3. Check if you don't see the Dax onboarding 
4. Check if FAQ was renamed to Debug
5. Check if the footer text in the profile screen is now in the header
